vtr_flow/misc/yosys/synthesis.tcl

```diff
@@ -66,7 +73,23 @@ techmap -map +/parmys/aldffe2dff.v
 
 opt -full
 
-parmys -a QQQ -nopass -c CCC YYY
+#parmys -a QQQ -nopass -c CCC YYY
+#parmys -a QQQ -c CCC YYY
+#memory -nomap
+#flatten
+
+
+# Separate opt for Parmys execution(verilog or system-verilog)
+if {$env(PARSER) == "default"} {
+    puts "Running Parmys with disables additional passes "
+    parmys -a QQQ -nopass -c CCC YYY
+
+} elseif {$env(PARSER) == "system-verilog" || $env(PARSER) == "surelog"} {
+    puts "Running Parmys with Additional Passes Resolve Conflicts"
```

Comment:
I'm not sure what this output means.  Wouldn't it be more clear to say "Running Parmys with additional passes to flatten all files and parse SystemVerilog" ?